src/plugins/bmqauthnpass/bmqauthnpass_passauthenticator.h

```diff
@@ -99,10 +101,16 @@ class PassAuthenticator : public mqbplug::Authenticator {
 
     bslma::Allocator* d_allocator_p;
 
-  public:
+  private:
     // NOT IMPLEMENTED
-    PassAuthenticator(const PassAuthenticator& other)            = delete;
-    PassAuthenticator& operator=(const PassAuthenticator& other) = delete;
+    PassAuthenticator(const PassAuthenticator& other) BSLS_KEYWORD_DELETED;
+    PassAuthenticator&
+    operator=(const PassAuthenticator& other) BSLS_KEYWORD_DELETED;
+
+  public:
+    // TRAITS
+    BSLMF_NESTED_TRAIT_DECLARATION(PassAuthenticationResult,
```

Comment:
```suggestion
    BSLMF_NESTED_TRAIT_DECLARATION(PassAuthenticator,
```
If the provided class name doesn't match the current class name the trait will not work